ledger/blockdb.go

```diff
@@ -208,6 +222,22 @@ func blockLatest(tx *sql.Tx) (basics.Round, error) {
 	return 0, fmt.Errorf("no blocks present")
 }
 
+// Returns amount of blocks in Block DB
+// At least one row (genesis block) must present, empty table is threated as error
+func blockCount(tx *sql.Tx) (uint64, error) {
+	var count sql.NullInt64
+	err := tx.QueryRow("SELECT COUNT(rnd) FROM blocks").Scan(&count)
```

Comment:
Using `SELECT COUNT(1)` should be faster.

Reply:
Switched to MIN instead